pkg/storage/utils/decomposedfs/node/node.go

```diff
@@ -435,26 +402,17 @@ func (n *Node) readOwner() (*userpb.UserId, error) {
 	}
 
 	// lookup IDP in extended attributes
-	attr, err = xattrs.Get(rootNodePath, xattrs.OwnerIDPAttr)
-	switch {
-	case err == nil:
-		owner.Idp = attr
-	case xattrs.IsAttrUnset(err):
-		// ignore
```

Comment:
Why don't we need this check any more?

Reply:
Hm, CI seems to be happy without it, but I removed this check intermittently during a bigger refactoring which I rolled back again, but I now agree that it should probably still be there. Let me try to add it back.